src/main/java/org/folio/rest/impl/EholdingsPackagesImpl.java

```diff
@@ -397,41 +449,49 @@ private String getResourceId(TitleResult titleResult) {
 
   private CompletableFuture<Void> deleteTags(PackageId packageId, String tenant) {
     return
-    packageRepository.delete(packageId, tenant)
-    .thenCompose(o -> tagRepository.deleteRecordTags(tenant, packageId.getProviderIdPart() + "-" + packageId.getPackageIdPart(), RecordType.PACKAGE))
-      .thenCompose(aBoolean ->  completedFuture(null));
+      packageRepository.delete(packageId, tenant)
+        .thenCompose(o -> tagRepository.deleteRecordTags(tenant, packageId.getProviderIdPart() + "-" + packageId.getPackageIdPart(), RecordType.PACKAGE))
+        .thenCompose(aBoolean -> completedFuture(null));
   }
 
-  private CompletableFuture<PackageResult> updateTags(PackageByIdData packageData, String tenant, Tags tags) {
+  private CompletableFuture<Void> updateTags(Tags tags, DbPackage dbPackage, String tenant) {
```

Comment:
After we update tags, are we not supposed to return updated tag list or empty list in case there are no tags? Is it right to return <Void> here?

Reply:
If update completed successfully then the new list of tags is the same as passed parameter, 
previously we were putting it into PackageResult to make it easier to chain future calls, but now this method is used in different places so the method caller constructs PackageResult

Reply:
got it, thanks